File: Custom_Alert_Dialog/app/src/main/java/com/example/custom_alert_dialog/MainActivity.java
package com.example.custom_alert_dialog;

import androidx.appcompat.app.AlertDialog;
import androidx.appcompat.app.AppCompatActivity;

import android.app.Dialog;
import android.content.DialogInterface;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

//        AlertDialog.Builder ad=new AlertDialog.Builder(this);
        Dialog dialog=new Dialog(this);
        dialog.setContentView(R.layout.custome_alert_dialog);

        dialog.show();
    }
}

File: RecyclerView_InKotlin/app/src/main/java/com/example/recyclerview_inkotlin/MainActivity.kt
package com.example.recyclerview_inkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.LinearLayout
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

private lateinit var recyclerView: RecyclerView;
private lateinit var adapter: Myadapter
private lateinit var  ModelList:MutableList<Model>
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        recyclerView =findViewById(R.id.recyclerview);
        recyclerView.layoutManager = LinearLayoutManager(this)

        val model=Model("Star","19","Engineer")
        val model2=Model("Isham","21","Banker")
        val model3=Model("Nootan","43","Teacher")

        ModelList= mutableListOf()

        ModelList.addAll(listOf(model,model2,model3))
        adapter= Myadapter(ModelList)
        recyclerView.adapter= adapter;


    }
}